Tests/SwiftParserTest/ExpressionTests.swift

```diff
@@ -668,7 +671,8 @@ final class ExpressionTests: XCTestCase {
         DiagnosticSpec(message: ##"expected '"""#' to end string literal"##, fixIts: [##"insert '"""#'"##])
       ],
       fixedSource: ##"""
-        #"""a"""#
+        #"""a
+        """#
```

Comment:
Here is a bug here too. 
We should in the parser make an error (If it's not already there) and apply a correct fix-it.